homeassistant/components/transmission/sensor.py

```diff
@@ -82,6 +91,12 @@ def available(self):
         """Could the device be accessed during the last update call."""
         return self._transmission_api.available
 
+    @property
+    def state_attributes(self):
```

Comment:
Please rename this to `device_state_attributes`.

Reply:
Thank you!  Is there a circumstance in which I should use state_attributes instead of device_state_attributes?

Reply:
The device facing integrations should only implement `device_state_attributes`. The base entity integrations implement `state_attributes`.